diff --git a/node_modules/quill/formats/formula.js b/node_modules/quill/formats/formula.js
index 1c1b1a5..e71af8b 100644
--- a/node_modules/quill/formats/formula.js
+++ b/node_modules/quill/formats/formula.js
@@ -26,7 +26,9 @@ class Formula extends Embed {
     const {
       formula
     } = this.value();
-    return `<span>${formula}</span>`;
+    const span = document.createElement('span');
+    span.textContent = formula;
+    return span.outerHTML;
   }
 }
 export default Formula;
diff --git a/node_modules/quill/formats/video.js b/node_modules/quill/formats/video.js
index 135a297..b3a4d51 100644
--- a/node_modules/quill/formats/video.js
+++ b/node_modules/quill/formats/video.js
@@ -41,7 +41,11 @@ class Video extends BlockEmbed {
     const {
       video
     } = this.value();
-    return `<a href="${video}">${video}</a>`;
+    const sanitizedVideo = Video.sanitize(video);
+    const a = document.createElement('a');
+    a.setAttribute('href', sanitizedVideo);
+    a.textContent = sanitizedVideo;
+    return a.outerHTML;
   }
 }
 export default Video;
